src/containerapp/azext_containerapp/_help.py

```diff
@@ -2366,3 +2366,52 @@
       text: |
           az containerapp env http-route-config delete -g MyResourceGroup -n MyEnvironment -r configname
 """
+
+helps['containerapp env premium-ingress show'] = """
+    type: command
+    short-summary: Show the premium ingress settings for the environment.
+    examples:
+    - name: Show the premium ingress settings for the environment.
+      text: |
+          az containerapp env premium-ingress show -g MyResourceGroup -n MyEnvironment
+"""
+
+helps['containerapp env premium-ingress'] = """
+    type: group
+    short-summary: Configures premium ingress settings for the environment.
```

Comment:
Please use the first-person voice in the help messae
```suggestion
    short-summary: Configure premium ingress settings for the environment.
```